test/image-prepare

```diff
@@ -208,6 +208,10 @@ def main():
     parser = argparse.ArgumentParser(
         description='Prepare testing environment, download images and build and install cockpit',
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('-e', '--env', metavar='NAME=VAL', action='append',
+                        help="Add a line to /etc/environment")
+    parser.add_argument('-d', '--debug', action='append_const', dest='env', const='COCKPIT_DEBUG=all',
+                        help="Equivalent to --env=COCKPIT_DEBUG=all")
```

Comment:
Note that this works for the Python bridge only, not the C bridge (that'd be `G_MESSAGES_DEBUG`)